xwayland/xwm.c

```diff
@@ -1518,8 +1541,30 @@ void wlr_xwayland_surface_restack(struct wlr_xwayland_surface *surface,
 	}
 	values[idx++] = mode;
 
-	xcb_configure_window(xwm->xcb_conn, surface->window_id, flags, values);
-	xcb_flush(xwm->xcb_conn);
```

Comment:
This has disappeared.

Reply:
Oops, nice catch -- an earlier version of this flushed within `xwm_set_net_client_list_stacking`, but no longer does.